changelog

```diff
@@ -0,0 +1,4 @@
+- set correct Travis CI setting
```

Comment:
No need for a changelog, the releases contains the changelog for now :)

Reply:
Oops! It's my fault. I will remove this.